docs/examples.rst

```diff
@@ -119,21 +125,8 @@ field must be added to store the directory of the file as follows:
     }
     ?>
 
-.. code:: php
-
-    <?php
-    /*
-       In the present example, these changes would be made in:
-       src/Template/Users/add.ctp
-       src/Template/Users/edit.ctp
-    */
-    ?>
-
-    <?php echo $this->Form->create($user, ['type' => 'file']); ?>
-        <?php echo $this->Form->input('username'); ?>
-        <?php echo $this->Form->input('photo', ['type' => 'file']); ?>
-        <?php echo $this->Form->input('photo_dir', ['type' => 'hidden']); ?>
-    <?php echo $this->Form->end(); ?>
```

Comment:
I removed it because the `photo_dir` input field is not required for deleting files, and because users should not be able to modify that value, and without that field the form is the same as in the previous example. 

I'd probably suggest to remove even more, and show only the different behavior configuration instead of the whole `UsersTable` class. But that's all personal preference, not sure what's best for the average reader. If you think it's better to provide a complete set of snippets, then I'll add it back in.